### PR TITLE
Refactor - #4316 - tC UI toolkit: Bookmark

### PR DESCRIPTION
**Pull request by richmahn**
Do what it says in the readme to start tc-ui-toolkit-test in a browser and see the wonderful Bookmark toggle in the VerseCheck (shows three components in the test page: CheckInfoCard, VerseCheck in select mode, and VerseCheck in view mode which has the Bookmark component).

I added enzyme to the tc-ui-toolkit packages for testing with mount and shallow.

---
This change is [Reviewable](https://reviewable.io/reviews/translationcoreapps/tc-ui-toolkit/11)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "start": "webpack --watch",
     "build": "webpack",
-    "setup": "npm i; npm run build; npm link; cd tc-ui-toolkit-test && npm i; npm link tc-ui-toolkit;"
+    "setup": "npm i && npm run build && npm link && cd tc-ui-toolkit-test && npm i && npm link tc-ui-toolkit"
```
Comment (richmahn) [inline package.json]: This was not working in Windows' Git Bash with semicolons, so used &&'s to make it work.

**Comment by mannycolon** [comment]
@richmahn The bookmark should also go inside the verse check component

**Comment by richmahn** [comment]
@mannycolon Ok, will add it there.

**Comment by richmahn** [comment]
@mannycolon Done. Now in VerseCheck and the test page shows VerseCheck in select mode and view mode with a working Bookmark toggle.

**Comment by mannycolon** [comment]
Reviewed 7 of 13 files at r1, 6 of 7 files at r3.
Review status: 13 of 14 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io/reviews/translationcoreapps/tc-ui-toolkit/11)*

**Comment by mannycolon** [comment]
Reviewed 1 of 13 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io/reviews/translationcoreapps/tc-ui-toolkit/11)*

**Comment by PhotoNomad0** [comment]
:LGTM:

**Comment by PhotoNomad0** [comment]
LGTM - testing and code review.

**Comment by RoyalSix** [comment]
Reviewed 7 of 13 files at r1, 6 of 7 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[testSetUp/enzyme.js, line 1 at r3](https://reviewable.io/reviews/translationcoreapps/tc-ui-toolkit/11#-LC5ksL66pdqO8kR1zAp:-LC5ksL66pdqO8kR1zAq:b-h3l68v) ([raw file](https://github.com/translationcoreapps/tc-ui-toolkit/blob/fafb19cf0b97ead5c228e483a48bd551ecb77d65/testSetUp/enzyme.js#L1)):*
> ```JavaScript
> import Enzyme from 'enzyme';
> ```

I thought that we don't need the shim anymore? Because we are using material-ui next and React 16?

---


*Comments from [Reviewable](https://reviewable.io/reviews/translationcoreapps/tc-ui-toolkit/11)*

**Pull request merged**